### DISCUSSION TITLE
Ability to ignore tables?

### DISCUSSION THREAD

**Discussion by nvitaterna** [prisma/prisma#13157] [Q&A] [answered]
I think this would be great for things like headless CMSs that alter the database with their own configuration tables. I've got something like [Directus](https://docs.directus.io/getting-started/quickstart/) playing nicely with Prisma (after pulling in their database to my schema and fiddling around with migrations)... But it would be best if there was a simple way to hide these tables from prisma as if they don't exist.

This use case comes out of wanting to manage content easily without having to build out an interface myself - I can make use of the CMS interface for editing content while using Prisma for everything else.

**Comment by janpio** [comment, marked as answer]
I think this is the feature request issue for this: https://github.com/prisma/prisma/issues/8864 Please leave a 👍 or comment with additional information if that is the case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
